config/services.yml

```diff
@@ -851,6 +851,31 @@ https://cmr.uat.earthdata.nasa.gov:
         - image/tiff
       reprojection: true
 
+  - name: net2cog
+    description: |
+      Converts NetCDF files to COG. Includes extension for running in NASA Harmony. https://github.com/podaac/net2cog
+    data_operation_version: '0.19.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/harmony/net2cog
+    umm_s: S1268612173-POCLOUD
+    capabilities:
+      subsetting:
+        variable: true
+      output_formats:
+        - image/tiff
+        # Don't know how specific/accuate this should be. This is (as far as I can tell) the 'official' mime-type that specifies COG. Got this value from https://github.com/opengeospatial/CloudOptimizedGeoTIFF/issues/1 and from pystac media type constant https://pystac.readthedocs.io/en/stable/api/media_type.html
+        - image/tiff; application=geotiff; profile=cloud-optimized
```

Comment:
Harmony ignores everything after the semicolon so this string will match against any user request that includes `image/tiff` like `image/tiff; profile=foo`. So you can delete this entry - maybe keep in a comment about the official COG mimetype in here if it's useful.